src/main/proto/service.proto

```diff
@@ -27,6 +27,8 @@ service FinApp {
   rpc CreateCustomer(Customer) returns (.google.protobuf.Empty) {}
 
   rpc CreateAccount(Account) returns (.google.protobuf.Empty) {}
+
+  rpc addAccountForCustomer(AddAccountForCustomerRequest) returns (.google.protobuf.Empty) {}
```

Comment:
https://developers.google.com/protocol-buffers/docs/style#message_and_field_names
```
Use CamelCase (with an initial capital) for message names – for example, SongServerRequest.
```

Reply:
In retrospect, `CreateCustomerRoleForAccount` might be a better name. What do you think?

Reply:
+1

Reply:
i changed the name to mimic the other requests, but lmk if i should change it (and maybe even the other requests to say explicitly `...Request`

Reply:
Perhaps file a new issue to do the rename to CreateCustomerRoleForAccount and return UUID as a separate PR.

However as part of this PR we should follow CamelCase for message names, i.e:
AddAccountForCustomer

Reply:
ok i filed this in #25